ocs_ci/cleanup/aws/cleanup.py

```diff
@@ -252,7 +252,7 @@ def aws_cleanup():
         help="""
             Maximum running time of the cluster (in hours).
             Clusters older than this will be deleted.
-            The minimum is 10 hours
+            The maximum is 40 hours
```

Comment:
This is not a maximum, it is a minimum value. This stops us from accidentally cleaning up every cluster older than say '4' hours.

Reply:
The way I am interpreting the semantic is "> 10 hours" is the maximum time the cluster can live in AWS?

Reply:
The minimum can be anything less than 10 hours?